codeforamerica/vita-min
# Add new FYST production and demo messaging service SIDs to environment credentials

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-167
## Is PM acceptance required? (delete one)
- Yes?

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- created new messaging service via Twilio console called 'FYST Production' and added short code to sender pool
- created new messaging service via Twilio console called 'FYST Demo' and added FYST 10DLC to sender pool
- re-structured Twilio section of environment credentials to have gyr and fyst sections
  - updated references to the gyr credentials around the code
## How to test?
- Acceptance testing should:
  - test sending SMS on heroku; look to see that the message sending status has updated (hub messages tab)
  - same test on demo once merged
  - same test on prod once deployed
- Risk Assessment
  - this touches the code related to GYR SMS so if we did it wrong, could break that
  - the only change should be that the credentials are nested differently so the references should now reflect that
  - we should generally keep an eye on GYR texting after this is deployed
